### PR TITLE
Uac business details

### PR DESCRIPTION
**Pull request by jdfox**
# Motivation and Context
Adding survey name and business name to survey message threads.

# What has changed
- added survey and business vars to conversation view template
- defined survey var from query to survey controller
- defined business var from existing conversation data
- updated existing test of thread, with mocking for survey name query
- also combined Makefile's unit and integration test targets into optional combined `make test` target

# How to test?
`make test`, deploy frontstage branch image to dev cluster and smoke test

# Links
- https://trello.com/c/oO3qMudG

# Screenshots (if appropriate):

### CODE REVIEW

File: tests/integration/test_secure_message.py [hunk 1]
```diff
@@ -45,13 +47,24 @@ def tearDown(self):
...
+                    "surveyMode": "EQ",
+                    "legalBasisRef": "STA1947"
+                })
 
         response = self.app.get("secure-message/threads/9e3465c0-9172-4974-a7d1-3a01592d1594",
                                 headers=self.headers, follow_redirects=True)
         self.assertEqual(response.status_code, 200)
         self.assertTrue('Peter Griffin'.encode() in response.data)
         self.assertTrue('testy2'.encode() in response.data)
         self.assertTrue('something else'.encode() in response.data)
```
Comment (warrenbailey) [inline tests/integration/test_secure_message.py:66]: This is missing tests checking for the survey name and business name which have been added to the page

**Review by warrenbailey — CHANGES_REQUESTED** [review]

File: tests/integration/test_secure_message.py (hunk 1 above)
Comment (jdfox) [reply tests/integration/test_secure_message.py:66]: survey name added to test

**Review by warrenbailey — APPROVED** [review]

**Pull request merged**